managers/NetworthManager.js

```diff
@@ -81,42 +59,21 @@ class NetworthManager {
         return this;
     }
 
-    /**
-     * Whether to only return the total networth or the items as well. Default: false
-     * @param {boolean} onlyNetworth
-     * @returns {NetworthManager} The NetworthManager instance
-     */
     setOnlyNetworth(onlyNetworth) {
         this.onlyNetworth = onlyNetworth;
         return this;
     }
 
-    /**
-     * Whether to stack items with the same attributes and price. Default: true
-     * @param {boolean} stackItems
-     * @returns {NetworthManager} The NetworthManager instance
-     */
     setStackItems(stackItems) {
         this.stackItems = stackItems;
         return this;
     }
 
-    /**
-     * Whether to include the item data as a property in the item object. Default: false
-     * @param {boolean} includeItemData
-     * @returns {NetworthManager} The NetworthManager instance
-     */
     setIncludeItemData(includeItemData) {
         this.includeItemData = includeItemData;
         return this;
     }
 
-    /**
-     * Manually update the items from the Hypixel API
-     * @param {number} [retries = 3] The amount of retries to fetch the items when failing to fetch them
-     * @param {number} [retryInterval = 1000] The interval in milliseconds to fetch the items from the Hypixel API when failing to fetch them
-     * @param {number} [currentRetry = 0] The current retry count
-     */
```

Comment:
Why remove all JSDocs?

Reply:
It imports types from .d.ts files, so no need for dupes